src/pages/chainlink-functions/tutorials/api-use-secrets.md

```diff
@@ -34,7 +34,7 @@ Apply [here](http://functions.chain.link/) to add your EVM account address to th
 ## Tutorial
 
 :::note[Complete the Before you begin section]
```

Comment:
State the goal directly in the note "title"

```suggestion
:::note[Set up your environment variables]
```

Reply:
Or just make it "Reminder" to be consistent with the other ones.

Reply:
Small improvement that's probably not an unrealistic lift for now: Single-source this reminder note itself.